web-ui/src/main/resources/catalog/js/jsonix/OWS_1_0_0.js

```diff
@@ -70,7 +70,7 @@ var OWS_1_0_0_Module_Factory = function() {
         col: true,
         mx: false,
         dom: false,
-        typed: false,
+        typed: true,
```

Comment:
I'm not an expert in JSONIX, but why is required to change a file in the JSONIX sources? No other way to manage this?

Reply:
i noticed this change on OWS_1_1_0, so duplicated it to OWS_1_0_0 and indeed after parsing works fine